src/constants/CdnConstants.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { Platform } from 'react-native'
+import RNFS from 'react-native-fs'
+
+import { type Theme } from '../types/Theme'
+
+// TODO: Add other CDN references to the theme files to allow third-party config:
+// flags, currency icons, partners, etc
+
+export const EDGE_CONTENT_SERVER_URL = 'https://content.edge.app'
+
+export const FLAG_LOGO_URL = `${EDGE_CONTENT_SERVER_URL}/country-logos`
+
+const directory = Platform.OS === 'ios' ? RNFS.DocumentDirectoryPath : RNFS.ExternalCachesDirectoryPath
+const BACKGROUND_IMAGE_FILE_NAME = 'login_bg.gif'
+export const BACKGROUND_IMAGE_URI = `${EDGE_CONTENT_SERVER_URL}/${BACKGROUND_IMAGE_FILE_NAME}`
+export const BACKGROUND_IMAGE_LOCAL_URI = `file://${directory}/${BACKGROUND_IMAGE_FILE_NAME}`
```

Comment:
End all identifiers with `URI` or `URL` to be consistent. I lean towards `URI` for some reason, but I have no conviction behind that. Here's some reasoning thought:

<img width="643" alt="Screen Shot 2022-04-19 at 15 21 05" src="https://user-images.githubusercontent.com/517469/164111187-48a73485-446d-452a-b2af-2248fbe0fdcd.png">

Reply:
My understanding is that URL describes a path/folder while the URI describes the actual resource

Reply:
Yeah you're right, all URI's are URL's, but not vice-versa